tests/shareRecipientTest.js

```diff
@@ -93,7 +69,33 @@ fdescribe('Main: Currently testing share recipient,', function () {
     })
   })
 
-  it('testing behavior : searching for users and groups', function (done) {
+  it('testing behavior : searching for users and groups', async function (done) {
+    await provider.addInteraction({
+      uponReceiving: 'a request to get share recipients (both users and groups)',
+      withRequest: {
+        method: 'GET',
+        path: Pact.Matchers.term({
+          matcher: '.*\\/ocs\\/v2\\.php\\/apps\\/files_sharing\\/api\\/v1\\/sharees$',
+          generate: '/ocs/v2.php/apps/files_sharing/api/v1/sharees'
+        }),
+        query: 'search=test&itemType=folder&page=1&perPage=200&format=json',
+        headers: validAuthHeaders
+      },
+      willRespondWith: {
+        status: 200,
+        headers: {
+          'Content-Type': 'application/json; charset=utf-8',
+          'Access-Control-Allow-Origin': origin
+        },
+        body: '{"ocs"' +
+          ':{"meta":{"status":"ok","statuscode":200,' +
+          '"message":"OK","totalitems":"","itemsperpage":""},' +
+          '"data":{"exact":{"users":[],"groups":[],"remotes":[]},' +
+          '"users":[{"label":"test123","value":{"shareType":0,"shareWith":"test123"}}],' +
+          '"groups":[{"label":"testGroup","value":{"shareType":1,"shareWith":"testGroup"}}],' +
```

Comment:
can we make the username and groupname dynamic

Reply:
fixed